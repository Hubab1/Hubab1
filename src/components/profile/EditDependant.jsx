import React, {Fragment} from 'react';
import styled from '@emotion/styled';

import { BackLink } from 'components/common/BackLink';
import { H1, H3 } from 'assets/styles';
import { ROUTES, RENTER_PROFILE_TYPE_CO_APPLICANTS } from 'app/constants';
import roommatesImage from 'assets/images/roommates.png';
import { InviteForm } from 'components/common/InviteForm';
import {connect} from "react-redux";
import { updateRenterProfile } from 'reducers/renter-profile';
//import { serializeDate } from 'utils/misc';

const SpacedH3 = styled(H3)`
    margin-top: 15px;
    margin-bottom: 30px;
`;

export class EditDependant extends React.Component {
    state = {confirmSent: false, errors: null};

    updateDependant = (values, { setSubmitting, setErrors }) => {
        this.props.history.push(`${ROUTES.PROFILE_OPTIONS}#${RENTER_PROFILE_TYPE_CO_APPLICANTS}`);
        setSubmitting(false);
        // const serialized = Object.assign({}, values);
        // serialized.birthday = serializeDate(serialized.birthday);
    };

    render () {
        if (this.props.profile == null) return null;

        const dependant = this.props.profile.dependents.find(x => x.id === parseInt(this.props.match.params.id));

        return (
            <Fragment>
                <H1>Edit a Person</H1>
                <SpacedH3>Enter their info below.</SpacedH3>
                <img src={roommatesImage} alt="hand with smartphone in it"/>
                <InviteForm
                    initialIsDependent={true}
                    dependantOnly={true}
                    initialValues={dependant}
                    onSubmitDependent={this.updateDependant}
                    handleOnSubmit={()=> {}}
                    displayedErrors={this.state.errors}
                    buttonText="Save Changes"
                />
                <BackLink to={`${ROUTES.PROFILE_OPTIONS}#${RENTER_PROFILE_TYPE_CO_APPLICANTS}`}/>
            </Fragment>
        );
    }
}
const mapStateToProps = state => ({
    profile: state.renterProfile,
});

export default connect(mapStateToProps, {updateRenterProfile})(EditDependant);
