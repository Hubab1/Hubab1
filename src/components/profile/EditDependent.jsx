import React, { Fragment } from 'react';
import styled from '@emotion/styled';
import PropTypes from 'prop-types';
import { BackLink } from 'components/common/BackLink';
import { H1, H3 } from 'assets/styles';
import { ROUTES, RENTER_PROFILE_TYPE_CO_APPLICANTS } from 'app/constants';
import roommatesImage from 'assets/images/roommates.png';
import { InviteForm } from 'components/common/InviteForm';
import { connect } from 'react-redux';
import { updateRenterProfile } from 'reducers/renter-profile';
import { serializeDate } from 'utils/misc';
import get from 'lodash/get';

const SpacedH3 = styled(H3)`
    margin-top: 15px;
    margin-bottom: 30px;
`;

export class EditDependent extends React.Component {
    state = { confirmSent: false, errors: null };

    updateDependant = (values, { setSubmitting, setErrors }) => {
        const serialized = Object.assign({}, values);
        const serializedBirthday = serializeDate(serialized.birthday);
        if (serializedBirthday) {
            serialized.birthday = serializedBirthday;
        }
        serialized.id = this.props.match.params.id;
        this.props
            .updateRenterProfile({ dependents: [serialized] })
            .then((res) => {
                if (res.errors) {
                    const errorsObj = get(res, 'errors.dependents');
                    const errors = errorsObj && Object.values(errorsObj)[0];
                    errors
                        ? setErrors(errors)
                        : this.setState({
                              errors: ['There was an error updating your dependent. Please Try again.'],
                          });
                } else {
                    this.props.history.push(`${ROUTES.PROFILE_OPTIONS}#${RENTER_PROFILE_TYPE_CO_APPLICANTS}`);
                }
                setSubmitting(false);
            })
            .catch(() => {
                this.setState({ errors: ['There was an error updating your dependent. Please Try again.'] });
                setSubmitting(false);
            });
    };

    render() {
        if (this.props.profile == null) return null;

        const dependent = this.props.profile.dependents.find((x) => x.id === parseInt(this.props.match.params.id));
        return (
            <Fragment>
                <H1>Edit a Person</H1>
                <SpacedH3>Enter their info below.</SpacedH3>
                <img src={roommatesImage} alt="hand with smartphone in it" />
                <InviteForm
                    initialIsDependent={true}
                    disableTypeChange={true}
                    initialValues={dependent}
                    onSubmitDependent={this.updateDependant}
                    handleOnSubmit={() => {}}
                    displayedErrors={this.state.errors}
                    buttonText="Save Changes"
                />
                <BackLink to={`${ROUTES.PROFILE_OPTIONS}#${RENTER_PROFILE_TYPE_CO_APPLICANTS}`} />
            </Fragment>
        );
    }
}

EditDependent.propTypes = {
    profile: PropTypes.object,
    updateRenterProfile: PropTypes.func,
    match: PropTypes.object,
    history: PropTypes.object,
};

const mapStateToProps = (state) => ({
    profile: state.renterProfile,
});

export default connect(mapStateToProps, { updateRenterProfile })(EditDependent);
