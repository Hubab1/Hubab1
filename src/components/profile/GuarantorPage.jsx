import React, { Fragment } from 'react';
import { connect } from 'react-redux';
import get from 'lodash/get';

import coin from 'assets/images/coin.png';
import { H1, SpacedH3 } from 'assets/styles';

import styled from '@emotion/styled';

import ConfirmationPage from 'components/common/ConfirmationPage/ConfirmationPage';
import { ROUTES, RENTER_PROFILE_TYPE_GUARANTOR } from 'app/constants';
import { selectors, fetchRenterProfile } from 'reducers/renter-profile';
import API from 'app/api';
import { InviteForm } from 'components/common/InviteForm';
import BackLink from 'components/common/BackLink';


const ImageContainer = styled.div`
    margin-top: 31px;
    margin-bottom: 31px;
    img {
        max-height: 90px;
        max-width: 90px;
    }
`

export class GuarantorPage extends React.Component {
    state = {confirmSent: false, errors: null};

    onSubmit = (values, { setSubmitting, setErrors }) => {
        return API.inviteGuarantor({guarantors: [values]}).then((res) => {
            setSubmitting(false);
            if (res.errors) {
                const errors = get(res, 'errors.guarantors[0]');
                if (errors) {
                    setErrors(errors);
                } else {
                    this.setState({errors: ['There was an error sending your guarantor an invite. Please Try again.']});

                }
            } else {
                this.props.fetchRenterProfile();
                this.setState({confirmSent: true});
            }
        }).catch((res) => {
            this.setState({errors: [res.errors]});
            setSubmitting(false);
        });
    }

    render () {
        if (this.state.confirmSent) {
            return <ConfirmationPage
                successMessage="Invite Sent!"
                secondarySuccessMessage="Youâ€™ll be able to check in on your guarantor's progress once you complete your application."
                buttonClick={()=>this.props.history.push(`${ROUTES.PROFILE_OPTIONS}#${RENTER_PROFILE_TYPE_GUARANTOR}`)}
                buttonText="Continue"
            />
        }
        return (
            <Fragment>
                <H1>Let's Invite a Guarantor</H1>
                <SpacedH3>Send your guarantor an application link via text or email.</SpacedH3>
                <ImageContainer>
                    <img src={coin} alt="coin"/>
                </ImageContainer>
                <InviteForm handleOnSubmit={this.onSubmit} displayedErrors={this.state.errors} />
                <BackLink to={`${ROUTES.PROFILE_OPTIONS}#${RENTER_PROFILE_TYPE_GUARANTOR}`}/>
            </Fragment>
        );
    }
}

const mapStateToProps = (state) => ({routes: selectors.selectOrderedRoutes(state)});

export default connect(mapStateToProps, {fetchRenterProfile})(GuarantorPage);
