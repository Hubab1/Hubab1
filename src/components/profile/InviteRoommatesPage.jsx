import React, { Fragment } from 'react';
import { Formik } from 'formik';
import * as Yup from 'yup';
import { Link } from 'react-router-dom';

import { H1, P, formContent, ErrorDetail } from 'assets/styles';
import inviteConfirm from 'assets/images/invite-confirm.png';
import FormTextInput from 'components/common/FormTextInput/FormTextInput';
import PhoneNumberInput from 'components/common/PhoneNumberInput';
import ActionButton from 'components/common/ActionButton/ActionButton';
import ConfirmationPage from 'components/common/ConfirmationPage/ConfirmationPage';
import { ROUTES } from 'app/constants';
import API from 'app/api';
import withRelativeRoutes from 'app/withRelativeRoutes';

export class InviteRoommatesPage extends React.Component {
    state = {confirmSent: false, errors: null};

    onSubmit = (values, { setSubmitting, setErrors }) => {
        API.inviteRoommate(values).then((res) => {
            setSubmitting(false);
            this.setState({confirmSent: true})
        }).catch((res) => {
            this.setState({errors: res.errors});
            setSubmitting(false);
        });
    }

    render () {
        if (this.state.confirmSent) {
            return <ConfirmationPage 
                successMessage="Invite Sent!"
                secondarySuccessMessage="You’ll be able to check in on your roommate’s progress once you complete your application."
                buttonClick={this.props._nextRoute}
                buttonText="Continue"
                secondaryButtonClick={() => this.setState({confirmSent: false})}
                secondaryButtonText="Add Another Roommate"
                confirmationImage={inviteConfirm}
            />
        } 
        return (
            <Fragment>
                <H1>Let's Invite Your Roommates</H1>
                <P>Tell us the basics and we’ll send them an invite to tell us the rest.</P>
                <Formik
                    validationSchema={Yup.object().shape({
                        first_name: Yup.string().required('First Name is required'),
                        last_name: Yup.string().required('Last Name is required'),
                        phone: Yup.string()
                            .required('Phone Number is required')
                            .matches(/^\(\d{3}\)\s\d{3}-\d{4}/, 'Must be a valid US phone number'),

                    })}
                    onSubmit={this.onSubmit}
                >
                    {({
                        values,
                        errors,
                        handleChange,
                        submitCount,
                        handleBlur,
                        handleSubmit,
                        isSubmitting
                    }) => (
                        <form onSubmit={handleSubmit} autoComplete="off">
                            <div className={formContent}>
                                <FormTextInput
                                    label="First Name"
                                    name="first_name"
                                    submitted={submitCount > 0}
                                    handleChange={handleChange}
                                    handleBlur={handleBlur}
                                    error={errors.first_name}
                                    value={values.first_name}
                                />
                                <FormTextInput
                                    label="Last Name"
                                    name="last_name"
                                    submitted={submitCount > 0}
                                    handleChange={handleChange}
                                    handleBlur={handleBlur}
                                    error={errors.last_name}
                                    value={values.last_name}
                                />
                                <PhoneNumberInput 
                                    label="Phone Number"
                                    name="phone"
                                    value={values.phone}
                                    handleChange={handleChange}
                                    error={submitCount > 0 && !!errors.phone}
                                    helperText={submitCount > 0 ? errors.phone : null}
                                />
                                <div>
                                    {!!this.state.errors && <ErrorDetail>{this.state.errors.error}</ErrorDetail>}
                                </div>
                                <ActionButton disabled={isSubmitting} marginTop="31px" marginBottom="10px">Send Invite</ActionButton>
                            </div>
                            <Link to={this.props._prev}>Go Back</Link>
                        </form>
                    )}
                </Formik>
            </Fragment>
        );
    }
}

export default withRelativeRoutes(InviteRoommatesPage, ROUTES.ROOMMATES);
