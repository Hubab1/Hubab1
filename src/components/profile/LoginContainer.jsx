import React, { Component } from 'react';

import { Bold, H1, BlueButton } from 'assets/index';
import Page from 'components/login/Page';

class LoginContainer extends Component {
  render() {
      return (
          <Page>
              <div>
                  <div>Jane Morgan</div>
                  <div></div>
                  <div>Leasing Agent</div>
                  <div>Sky Residencies</div>
              </div>
              <div>
                  <H1>Hi Sam,</H1>
                  <br/>
                  <br/>
                  <div>Your magically simple rental application starts here.</div>
                  <br/>
                  <div>Your new home awaits at <Bold>555 Waverly 605 W 11th St, New York, Unit 3F</Bold> awaits.</div>
                  <BlueButton>Click here</BlueButton>
              </div>
          </Page>
      );
  }
};

export default LoginContainer;
