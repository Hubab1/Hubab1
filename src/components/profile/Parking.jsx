import React from 'react';
import { connect } from 'react-redux';
import styled from '@emotion/styled';
import { Formik } from 'formik';
import Box from '@material-ui/core/Box';

import ActionButton from 'components/common/ActionButton/ActionButton';
import { BackLink } from 'components/common/BackLink';
import ItemAdder from 'components/common/ItemAdder';
import parkingImage from 'assets/images/parking.png';
import { H1, SpacedH3 } from 'assets/styles';
import { ROUTES, RENTER_PROFILE_TYPE_PARKING } from 'app/constants';
import { updateRenterProfile } from 'reducers/renter-profile';
import { rentalOptionsInitialValues } from 'utils/misc';
import PriceBreakdown from "./options/PriceBreakdown";


const ImageContainer = styled.div`
    margin-top: 31px;
    margin-bottom: 31px;
    img {
        max-height: 123px;
        max-width: 123px;
    }
`;

export const Parking = props => {
    if (!props.config || !props.application) return null;

    const onSubmit = (values, { setSubmitting, setErrors }) => {
        const selectedRentalOptionsArray = [];
        Object.entries(values).forEach(option => {
            selectedRentalOptionsArray.push({ rental_option: {id: parseInt(option[0])}, quantity: option[1]});
        });
        const selectedRentalOptions = Object.assign({}, {selected_rental_options: selectedRentalOptionsArray});

        return props.updateRenterProfile(selectedRentalOptions).then((res) => {
            if (res.errors) {
                setErrors(res.errors);
            } else {
                props.history.push(`${ROUTES.PROFILE_OPTIONS}#${RENTER_PROFILE_TYPE_PARKING}`);
            }
            setSubmitting(false);
        });
    };

    const getPriceBreakdown = (selectedOptions) => {
        return (
            <PriceBreakdown
                selectedOptions={selectedOptions}
                application={props.application}
            />);
    };

    const initialParkingOptions = props.application.selected_rental_options.parking;
    const parkingOptions = props.config.rental_options.parking || [];
    return <>
        <H1>Parking</H1>
        <SpacedH3>This is a request for parking. All parking is based on availability.</SpacedH3>
        <ImageContainer>
            <img src={parkingImage} alt="car parking"/>
        </ImageContainer>
        <Formik
            onSubmit={onSubmit}
            initialValues={rentalOptionsInitialValues(initialParkingOptions)}
        >
            {({
                values,
                handleSubmit,
                setFieldValue,
            }) => (
                <form className="text-left" onSubmit={handleSubmit} autoComplete="off">
                    { parkingOptions.map(option =>
                        <ItemAdder
                            key={option.id}
                            title={option.name}
                            subtitle={`$${option.monthly_amount}/mo per parking space`}
                            value={values[option.id]}
                            limit={option.limit}
                            onChange={e => setFieldValue(option.id, e)}
                        />
                    )}
                    {getPriceBreakdown(values)}
                    <ActionButton>Add Parking</ActionButton>
                </form>
            )}
        </Formik>
        <Box padding="20px">
            <BackLink to={`${ROUTES.PROFILE_OPTIONS}#${RENTER_PROFILE_TYPE_PARKING}`}/>
        </Box>
    </>;
};

const mapStateToProps = state => ({
    config: state.configuration,
    application: state.renterProfile,
});

export default connect(mapStateToProps, { updateRenterProfile })(Parking);
