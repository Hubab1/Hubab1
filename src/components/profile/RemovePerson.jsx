import React from 'react';
import styled from '@emotion/styled';
import { H1, H3, P, Bold } from 'assets/styles';
import {
    ROUTES,
    RENTER_PROFILE_TYPE_CO_APPLICANTS,
    RENTER_PROFILE_TYPE_DEPENDENT,
    RENTER_PROFILE_TYPE_GUARANTOR,
} from 'app/constants';
import GenericFormMessage from 'components/common/GenericFormMessage';
import ActionButton from 'components/common/ActionButton/ActionButton';
import { connect } from 'react-redux';
import API from 'app/api';
import { fetchRenterProfile } from 'reducers/renter-profile';
import PropTypes from 'prop-types';

const SkinnyH1 = styled(H1)`
    width: 70%;
`;

const SpacedH3 = styled(H3)`
    margin-top: 15px;
    margin-bottom: 22px;
`;

const CapitalizedSpan = styled.span`
    text-transform: capitalize;
`;

const Divider = styled.hr`
    border-style: none;
    border-bottom: 2px solid #eeeeee;
    margin-bottom: 22px;
`;

const Content = styled.div`
    text-align: left;
`;

export class RemovePerson extends React.Component {
    state = { errorSubmitting: false, financialSource: null, submitting: false };

    get returnLink() {
        return `${ROUTES.PROFILE_OPTIONS}#${RENTER_PROFILE_TYPE_CO_APPLICANTS}`;
    }

    get person() {
        if (!this.props.profile) return null;

        const {
            match: {
                params: { type, id },
            },
        } = this.props;
        const { profile } = this.props;

        switch (type) {
            case RENTER_PROFILE_TYPE_DEPENDENT:
                return profile.dependents.find((x) => x.id === parseInt(id));
            case RENTER_PROFILE_TYPE_CO_APPLICANTS:
                return profile.co_applicants.find((x) => x.id === parseInt(id));
            case RENTER_PROFILE_TYPE_GUARANTOR:
                // TODO: get the guarantors directly from the profile
                return profile.primary_applicant.guarantors.find((x) => x.id === parseInt(id));
            default:
                return null;
        }
    }

    onSubmit = async () => {
        const {
            match: {
                params: { type, id },
            },
        } = this.props;

        this.setState({ submitting: true });
        try {
            if (type === RENTER_PROFILE_TYPE_DEPENDENT) {
                await API.deletePerson(id);
            } else {
                await API.deleteInvitee(id);
            }

            this.props.fetchRenterProfile();
            this.setState({ submitting: false });
            this.props.history.push(this.returnLink);
        } catch {
            this.setState({ submitting: false, errorSubmitting: true });
        }
    };

    onCancel = () => {
        this.props.history.push(this.returnLink);
    };

    render() {
        if (!this.props.profile || !this.person) return null;

        const { match } = this.props;
        const person = this.person;
        const role = match.params.type === RENTER_PROFILE_TYPE_CO_APPLICANTS ? 'co-applicant' : match.params.type;
        const personLabel = role === RENTER_PROFILE_TYPE_GUARANTOR ? 'Guarantor' : 'Person';

        return (
            <>
                <SkinnyH1>Remove {personLabel}</SkinnyH1>
                <SpacedH3>
                    <CapitalizedSpan>
                        {person.first_name} {person.last_name}
                    </CapitalizedSpan>
                </SpacedH3>
                <Divider />
                <Content>
                    {this.state.errorSubmitting && (
                        <GenericFormMessage
                            type="error"
                            messages={['Oops! We had some trouble removing this person. Try again in a little bit.']}
                        />
                    )}
                    <Bold fontSize={18}>{`Are you sure you want to remove this ${personLabel.toLowerCase()}?`}</Bold>
                    <br />
                    <br />
                    {role === RENTER_PROFILE_TYPE_GUARANTOR ? (
                        <P>
                            You&apos;re about to remove <CapitalizedSpan>{person.first_name}</CapitalizedSpan> as
                            guarantor. Removing a guarantor prevents them from being able to financially back your lease
                            application.
                        </P>
                    ) : (
                        <P>
                            You&apos;re about to remove <CapitalizedSpan>{person.first_name}</CapitalizedSpan>
                            {`. Removing a person prevents them from being able to apply for this unit as a ${role} or from being added to the lease.`}
                        </P>
                    )}
                </Content>
                <ActionButton
                    id="submit-btn"
                    disabled={this.state.submitting}
                    onClick={this.onSubmit}
                    marginBottom={20}
                    marginTop={100}
                >
                    Remove {personLabel}
                </ActionButton>
                <ActionButton id="cancel-btn" onClick={this.onCancel} variant="outlined" marginBottom={20}>
                    Cancel
                </ActionButton>
            </>
        );
    }
}

RemovePerson.propTypes = {
    profile: PropTypes.object,
    match: PropTypes.object,
    fetchRenterProfile: PropTypes.func,
    history: PropTypes.object,
};

const mapStateToProps = (state) => ({
    profile: state.renterProfile,
});

export default connect(mapStateToProps, { fetchRenterProfile })(RemovePerson);
