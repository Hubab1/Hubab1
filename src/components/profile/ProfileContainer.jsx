import React, { Component } from 'react';
import { Route, Switch } from 'react-router-dom';

import { ROUTES } from 'app/constants';
import RenterProfileOptions from 'components/profile/options/RenterProfileOptions';
import GuarantorPage from 'components/profile/options/GuarantorPage';
import InviteRoommatesPage from 'components/profile/InviteRoommatesPage';
import PetsPage from 'components/profile/pets/PetsPage';

class ProfileContainer extends Component {
    render() {
        return (
            <Switch>
                <Route path={ROUTES.PROFILE_OPTIONS} component={RenterProfileOptions} />
                <Route path={ROUTES.GUARANTOR} component={GuarantorPage} />
                <Route path={ROUTES.ROOMMATES} component={InviteRoommatesPage} />
<<<<<<< HEAD
                <Route path={ROUTES.PETS} component={PetsPage} />
=======
>>>>>>> master
            </Switch>
        );
    }
};

export default ProfileContainer;
