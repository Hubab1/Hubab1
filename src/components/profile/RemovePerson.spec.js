import React from 'react';
import { shallow } from 'enzyme';

import mockConfig from 'reducers/mock-config.json';
import mockApplication from 'reducers/mock-profile.json';
import mockProfile from 'reducers/mock-profile.json';
import { RemovePerson } from 'components/profile//RemovePerson';


let defaultProps, fetchRenterProfile;

beforeEach(() => {
    fetchRenterProfile = jest.fn().mockResolvedValue({});
    defaultProps = {
        fetchRenterProfile: fetchRenterProfile,
        profile: {
            ...mockProfile,
            dependents: [{id: 1, first_name: 'John', last_name: 'Doe', birthday: null}]
        },
        application: mockApplication,
        config: mockConfig,
    }
});

it('matches snapshot: case dependent', function() {
    let wrapper = shallow(<RemovePerson
        {...defaultProps}
        match={{params: {id: 1, type: 'dependent'}, isExact: true, path: "", url: ""}}
    /> );

    expect(wrapper.getElement()).toMatchSnapshot();
});