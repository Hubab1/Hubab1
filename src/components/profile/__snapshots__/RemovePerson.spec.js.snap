// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot: case dependent 1`] = `
<React.Fragment>
  <ForwardRef(render)>
    Remove 
    Person
  </ForwardRef(render)>
  <ForwardRef(render)>
    John Doe
  </ForwardRef(render)>
  <ForwardRef(render) />
  <ForwardRef(render)
    fontSize={18}
  >
    Are you sure you want to remove this person?
  </ForwardRef(render)>
  <br />
  <br />
  <ForwardRef(render)>
    You're about to remove John. Removing a person prevents them from being able to apply for this unit as a dependent or from being added to the lease.
  </ForwardRef(render)>
  <ActionButton
    color="primary"
    disabled={false}
    id="submit-btn"
    marginBottom={20}
    marginTop={100}
    onClick={[Function]}
    successGreen={false}
    variant="contained"
  >
    Remove 
    Person
  </ActionButton>
  <ActionButton
    color="primary"
    id="cancel-btn"
    marginBottom={20}
    onClick={[Function]}
    successGreen={false}
    variant="outlined"
  >
    Cancel
  </ActionButton>
</React.Fragment>
`;
